File: bruteforce.py
import string
import time
import random

target = "Hello world"
letters = string.ascii_letters + " "
result = ""


for letter in target:
    while True:
        i = random.choice(letters)
        print (result + i) #Prints current state of result plus random letter

        if i == letter:
            result += i #Concantenating the correct letter to the result
            break
        time.sleep(0.01)
        

File: bruteforce.js
const target = "Hello world";
const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ ";
let result = "";

function randomCharacter() {
    return letters.charAt(Math.floor(Math.random() * letters.length));
}

async function bruteForce() {
    for (let letter of target) {
        while (true) {
            const i = randomCharacter();
            console.log(result + i); // Prints current state of result plus random letter

            if (i === letter) {
                result += i; // Concatenating the correct letter to the result
                break;
            }
            await new Promise(resolve => setTimeout(resolve, 10)); // Sleep for 10ms
        }
    }
}

bruteForce();
